src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -36,6 +36,29 @@
      * София - 900
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String inform = "";
+        String[] string;
+
+        if (Integer.parseInt(dateFrom.replaceAll("[^0-9]",""))
```

Comment:
@Kokhanevich what if you will compare dateFrom = 14.07.2019 and dateTo = 10.08.2019?